lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -681,7 +682,7 @@ postTx
     -> m (HTTP.Status, Either RequestException (ApiTransaction n))
 postTx ctx (wSrc, postTxEndp, pass) wDest amt = do
     addrs <- listAddresses @n ctx wDest
-    let destination = (addrs !! 1) ^. #id
+    let destination = (addrs !! 1) ^. (#address . #id)
```

Comment:
This nesting seems _wrong_ :thinking: 
(I'll comment on the swagger schema modification directly).